datasets/standbedrijven/dataset.json

```
@@ -0,0 +1,572 @@
+            "description": "De naam van de wijk"
+          },
+          "ggwgebiedIdentificatie": {
+            "type": "string",
+            "title": "ggwgebiedIdentificatie",
+            "shortname": "ggwId",
+            "description": "Unieke identificatie van het gebiedsgericht werken gebied"
+          },
+          "ggwgebiedVolgnummer": {
+            "type": "integer",
+            "title": "ggwgebiedVolgnummer",
+            "shortname": "ggwVolgnummer",
+            "description": "Uniek volgnummer van de toestand van het gebiedsgericht werken gebied"
+          },
+          "ggwgebiedCode": {
+            "type": "string",
+            "title": "ggwgebiedCode",
+            "shortname": "ggwCode",
+            "description": "Officiële code van het gebiedsgericht werken gebied"
+          },
+          "ggwgebiedNaam": {
+            "type": "string",
+            "title": "ggwgebiedNaam",
+            "shortname": "ggwNaam",
+            "description": "De naam van het gebiedsgericht werken gebied"
+          },
+          "stadsdeelIdentificatie": {
+            "type": "string",
+            "title": "stadsdeelIdentificatie",
+            "shortname": "sdlId",
+            "description": "Unieke identificatie van het stadsdeel"
+          },
+          "stadsdeelVolgnummer": {
+            "type": "integer",
+            "title": "stadsdeelVolgnummer",
+            "shortname": "sdlVolgnummer",
+            "description": "Uniek volgnummer van de toestand van het stadsdeel"
+          },
+          "stadsdeelCode": {
+            "type": "string",
+            "title": "stadsdeelCode",
+            "shortname": "sdlCode",
+            "description": "Officiële code van het stadsdeel"
+          },
+          "stadsdeelNaam": {
+            "type": "string",
+            "title": "stadsdeelNaam",
+            "shortname": "sdlNaam",
+            "description": "De naam van het stadsdeel"
+          },
+          "indicatieHandmatigeOpvoer": {
+            "type": "boolean",
+            "title": "indicatieHandmatigeOpvoer",
+            "shortname": "indHandmatigOpvoer",
+            "description": "Indicatie die aangeeft of een vestiging handmatig is toegevoegd aan het vestigingenregister in het programma VR-Beheer"
+          },
+          "typeMutatieCode": {
+            "type": "string",
+            "title": "typeMutatieCode",
+            "description": "Omschrijving die aangeeft hoe de vestiging is gemuteerd ten opzichte van de vorige stand"
+          }
+        },
+        "typeMutatieOmschrijving": {
+          "type": "string",
+          "title": "typeMutatieOmschrijving",
+          "description": "Omschrijving die aangeeft hoe de vestiging is gemuteerd ten opzichte van de vorige stand"
+        },
+        "peildatumVorigeStand": {
+          "type": "string",
+          "format": "date",
+          "title": "peildatumVorigeStand",
+          "description": "Peildatum van de vorige stand"
+        }
+      },
```

Deze staan een niveautje te hoog.

```suggestion
          },
          "typeMutatieOmschrijving": {
            "type": "string",
            "title": "typeMutatieOmschrijving",
            "description": "Omschrijving die aangeeft hoe de vestiging is gemuteerd ten opzichte van de vorige stand"
          },
          "peildatumVorigeStand": {
            "type": "string",
            "format": "date",
            "title": "peildatumVorigeStand",
            "description": "Peildatum van de vorige stand"
          }
        }
      },
```